python-sdk/nuscenes/map_expansion/fix_map.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import json
+import os
+
+from nuscenes.map_expansion.map_api import NuScenesMap
+from nuscenes.map_expansion.utils import drop_disconnected_lanes
+
+
+def fix_map(dataroot: str, map_name: str, out_suffix: str) -> None:
+    """
+    Fix a map file by dropping disconnected lanes.
+    Note that we do not recommend the use of this script as even disconnected lanes have useful information.
+
+    :param dataroot: Path of the nuScenes dataset.
+    :param map_name: Which map out of `singapore-onenorth`, `singepore-hollandvillage`, `singapore-queenstown` and
+        'boston-seaport'.
+    :param out_suffix: A suffix to the output file name, e.g. "fixed" results in "<map>_fixed.json".
+    """
+    # Load map.
+    nusc_map = NuScenesMap(map_name=map_name, dataroot=dataroot)
+
+    # Drop disconnected lanes.
+    nusc_map = drop_disconnected_lanes(nusc_map)
```

Comment:
@holger-nutonomy I don't know if it makes sense to drop disconnected lanes. All but one of these still has centerline geometry info, which can be very useful.

Reply:
@emwolff : the instructions at the top of this script literally says that "we recommend that you don't run this script". haha. Perhaps it's better to just delete it?

Reply:
Okay. Removing it.